pkg/api/api.go

```diff
@@ -61,7 +61,10 @@ type API struct {
 	TweakResponseFunc func(map[string]any)
 	// Parent is used for SubPath APIs to store information about the configuration type and ID of the related
 	// configuration once Resolved() is called.
-	Parent *API
+
+	// CheckEqualFunc can be used to compare existing objects with current objects based on their payload
+	CheckEqualFunc func(map[string]any, map[string]any) bool
+	Parent         *API
```

Comment:
you've accidentally removed this field from its comment - should stay on line 64